automation/src/test/scala/org/broadinstitute/dsp/consent/services/DarService.scala

```diff
@@ -0,0 +1,119 @@
+        dataMap += ("linkedIn" -> JsString(researcherInfo.linkedIn.getOrElse("")))
+        dataMap += ("researcherGate" -> JsString(researcherInfo.researcherGate.getOrElse("")))
+        dataMap += ("orcid" -> JsString(researcherInfo.orcid.getOrElse("")))
+        dataMap += ("institution" -> JsString(researcherInfo.institution.getOrElse("")))
+        dataMap += ("department" -> JsString(researcherInfo.department.getOrElse("")))
+        dataMap += ("division" -> JsString(researcherInfo.division.getOrElse("")))
+        dataMap += ("address1" -> JsString(researcherInfo.address1.getOrElse("")))
+        dataMap += ("address2" -> JsString(researcherInfo.address2.getOrElse("")))
+        dataMap += ("city" -> JsString(researcherInfo.city.getOrElse("")))
+        dataMap += ("zipCode" -> JsString(researcherInfo.zipcode.getOrElse("")))
+        dataMap += ("country" -> JsString(researcherInfo.country.getOrElse("")))
+        dataMap += ("state" -> JsString(researcherInfo.state.getOrElse("")))
+        dataMap += ("piName" -> JsString(researcherInfo.piName.getOrElse("")))
+        dataMap += ("academicEmail" -> JsString(researcherInfo.academicEmail.getOrElse("")))
+        dataMap += ("piEmail" -> JsString(researcherInfo.piEmail.getOrElse("")))
+        dataMap += ("isThePi" -> JsString(researcherInfo.isThePI.getOrElse("")))
+        dataMap += ("havePi" -> JsString(researcherInfo.havePI.getOrElse("")))
+        dataMap += ("pubmedId" -> JsString(researcherInfo.pubmedID.getOrElse("")))
+        dataMap += ("scientificUrl" -> JsString(researcherInfo.scientificURL.getOrElse("")))
+        dataMap += ("userId" -> JsNumber(userId))
+        dataMap += ("itDirector" -> JsString("Test IT Director"))
+        dataMap += ("signingOfficial" -> JsString("Test Signing Official"))
+        dataMap += ("checkCollaborator" -> JsBoolean(false))
+        dataMap += ("anvilUse" -> JsBoolean(false))
+        dataMap += ("cloudUse" -> JsBoolean(false))
+        dataMap += ("localUse" -> JsBoolean(true))
+        dataMap += ("projectTitle" -> JsString("Test Project"))
+        dataMap += ("rus" -> JsString("test"))
+        dataMap += ("nonTechRus" -> JsString("test non technical"))
+        dataMap += ("hmb" -> JsBoolean(false))
+        dataMap += ("poa" -> JsBoolean(true))
+        dataMap += ("other" -> JsBoolean(false))
+        dataMap += ("forProfit" -> JsBoolean(false))
+        dataMap += ("oneGender" -> JsBoolean(false))
+        dataMap += ("pediatric" -> JsBoolean(false))
+        dataMap += ("illegalBehavior" -> JsBoolean(false))
+        dataMap += ("addiction" -> JsBoolean(false))
+        dataMap += ("sexualDiseases" -> JsBoolean(false))
+        dataMap += ("stigmatizedDiseases" -> JsBoolean(false))
+        dataMap += ("vulnerablePopulation" -> JsBoolean(false))
+        dataMap += ("populationMigration" -> JsBoolean(false))
+        dataMap += ("psychiatricTraits" -> JsBoolean(false))
+        dataMap += ("notHealth" -> JsBoolean(false))
```

Comment:
I think this is an example of a mutability that makes sense 👍🏽

Reply:
Thank you 👍